### PR TITLE
[Feat]:Added checkbox and radio input filter functionality

### PR DESCRIPTION
**Pull request by t007rushi**
3 more Filters were added:
1 with checkbox input
2 with radio inputs

recording

<details><summary>Browser metadata</summary>

```
Path:      /products
Browser:   Chrome 99.0.4844.74 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-5--game-kart-ecom.netlify.app%2Fproducts&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-5--game-kart-ecom.netlify.app/products) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=623822922bd3c10008b69c67&commentID=623823e726e06310fdb5dd75&resolution=resolved)

### CODE REVIEW

File: src/Context/filter-context.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
-import { CLEAN_UP, PRICERANGE, SORT } from "../constant/filterConstant";
+import {
+  CLEAN_UP,
+  CONDITION,
+  PRICERANGE,
+  RATING,
+  SORT,
+} from "../constant/filterConstant";
 
 const FilterContext = createContext(null);
```
Comment (rkapoor10) [inline src/Context/filter-context.js:10]: empty object instead of null

File: src/Context/products-context.js [hunk 2]
```diff
@@ -1,17 +1,22 @@
-import data from "../data";
 import { createContext, useContext, useState } from "react";
+import axios from "axios";
 
-const ProductContext = createContext(null)
+const ProductContext = createContext(null);
```
Comment (rkapoor10) [inline src/Context/products-context.js:4]: check this also

File: src/components/Input/Input.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./input.css";
+import { useFilter } from "../../Context/filter-context";
+
+const Input = ({ inputs }) => {
+  const { dispactherforfilter } = useFilter();
```
Comment (MarkVed17) [inline src/components/Input/Input.jsx:6]: Use the proper name. I think you expected "dispatcherforfilter" here.

File: src/components/SelectInput/SelectInput.jsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ const SelectInput = () => {
           dispactherforfilter({ type: SORT, payload: e.target.value })
```
Comment (rkapoor10) [inline src/components/SelectInput/SelectInput.jsx:14]: liked this function as it removes hardcode

File: src/filter-data.js [hunk 5]
```diff
@@ -1,77 +1,86 @@
-const categ = [
```
Comment (rkapoor10) [inline src/filter-data.js:1]: Liked that you created an object and made filters through input component

**Review by rkapoor10 — APPROVED** [review]
This code looks good to merge.

**Review by MarkVed17 — APPROVED** [review]
The code looks good. 
A suggestion: follow some better naming practices.

File: src/components/Input/Input.jsx (hunk 3 above)
Comment (t007rushi) [reply src/components/Input/Input.jsx:6]: Thank you for pointing out typo

File: src/Context/filter-context.js (hunk 1 above)
Comment (t007rushi) [reply src/Context/filter-context.js:10]: Yes, will do that thanks for sugesstion

**Pull request merged**